teamcity: failed tests on master: test/TestLogic

The following tests appear to have failed:

[#657929](https://teamcity.cockroachdb.com/viewLog.html?buildId=657929):


```
--- FAIL: test/TestLogic/distsql-metadata/cascade/UpdateCascade_CompositeFKs (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/5node-distsql-disk/distsql_numtables (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql-disk/distsql_numtables
--- FAIL: test/TestLogic/distsql/explain (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/explain
--- FAIL: test/TestLogic/distsql-metadata/name_escapes (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/name_escapes
--- FAIL: test/TestLogic/default/default (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/default
--- FAIL: test/TestLogic/distsql/snapshot_certain_read (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/snapshot_certain_read
--- FAIL: test/TestLogic/distsql-metadata/cascade/UpdateSetNull_Basic1 (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/alter_table with config distsql-metadata: 165 tests, 0 failures
--- total progress: 28977 statements/queries
--- FAIL: test/TestLogic/default/distinct_on (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/distinct_on
--- FAIL: test/TestLogic/default/function_lookup (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/function_lookup
--- FAIL: test/TestLogic/default/insert (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/insert
--- FAIL: test/TestLogic/default/builtin_function (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/builtin_function
--- FAIL: test/TestLogic/distsql-metadata/snapshot_issue2861 (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/snapshot_issue2861
--- FAIL: test/TestLogic/distsql-metadata/generators/nested_SRF (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/cascade/UpdateCascade_TwoUpdatesReverse (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/alter_table: 145 statements/queries
--- FAIL: test/TestLogic/distsql/check_constraints (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/check_constraints
--- FAIL: test/TestLogic/distsql-metadata/float (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/float
--- FAIL: test/TestLogic/default/cascade/DeleteCascade_CompositeFKs (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/array with config default: 233 tests, 0 failures
--- FAIL: test/TestLogic/default/cascade/UpdateCascade_Interleaved (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/UpdateCascade_TwoUpdates (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/cascade/DeleteSetDefault_ToUpdateCascadeNotNull (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/rename_sequence (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/rename_sequence
--- FAIL: test/TestLogic/parallel-stmts/postgres_jsonb (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/postgres_jsonb
--- FAIL: test/TestLogic/distsql-metadata/poison_after_push (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/poison_after_push
--- FAIL: test/TestLogic/default/scatter (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/scatter
--- FAIL: test/TestLogic/parallel-stmts/statement_source (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/statement_source
--- FAIL: test/TestLogic/parallel-stmts/update (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/update
--- FAIL: test/TestLogic/parallel-stmts/cascade/DeleteSetDefault_Basic2 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/cascade (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/cascade
--- FAIL: test/TestLogic/distsql/inverted_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/inverted_index
--- FAIL: test/TestLogic/5node-distsql-disk/distsql_distinct_on (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql-disk/distsql_distinct_on
--- FAIL: test/TestLogic/default/alter_table (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/alter_table
--- FAIL: test/TestLogic/distsql/cascade/DeleteSetDefault_ToUpdateCascade (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/json (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/json
--- FAIL: test/TestLogic/distsql-metadata/txn (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/txn
--- FAIL: test/TestLogic/parallel-stmts/cascade/DeleteSetDefault_Basic1_WrongDefault (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/scrub (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/scrub
--- FAIL: test/TestLogic/distsql/generators/24866 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/parallel-stmts/with (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/with
=== CONT  TestLogic/parallel-stmts/serializable_eager_restart
=== CONT  TestLogic/parallel-stmts/returning
=== CONT  TestLogic/parallel-stmts/serial
=== CONT  TestLogic/parallel-stmts/sequences
=== CONT  TestLogic/parallel-stmts/select_non_covering_index_filtering
=== CONT  TestLogic/parallel-stmts/select_non_covering_index
=== CONT  TestLogic/parallel-stmts/select_index_hints
=== CONT  TestLogic/parallel-stmts/select_index
--- done: testdata/logic_test/select_non_covering_index with config parallel-stmts: 18 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/select
--- done: testdata/logic_test/serial with config parallel-stmts: 11 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/scrub
--- done: testdata/logic_test/returning with config parallel-stmts: 5 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/schema_change_retry
--- done: testdata/logic_test/select_non_covering_index_filtering with config parallel-stmts: 10 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/scale
--- done: testdata/logic_test/serializable_eager_restart with config parallel-stmts: 11 tests, 0 failures
--- done: testdata/logic_test/select_index_hints with config parallel-stmts: 28 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/run_control
=== CONT  TestLogic/parallel-stmts/time
--- done: testdata/logic_test/run_control with config parallel-stmts: 28 tests, 0 failures
--- total progress: 8717 statements/queries
=== CONT  TestLogic/parallel-stmts/with
--- done: testdata/logic_test/time with config parallel-stmts: 59 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/window
--- done: testdata/logic_test/scale with config parallel-stmts: 50 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/where
--- done: testdata/logic_test/schema_change_retry with config parallel-stmts: 12 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/views
--- done: testdata/logic_test/scrub with config parallel-stmts: 32 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/uuid
--- done: testdata/logic_test/select with config parallel-stmts: 107 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/upsert
--- done: testdata/logic_test/with with config parallel-stmts: 19 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/update
--- done: testdata/logic_test/where with config parallel-stmts: 19 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/txn
--- done: testdata/logic_test/window with config parallel-stmts: 167 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/tuple
--- done: testdata/logic_test/uuid with config parallel-stmts: 29 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/truncate
--- progress: testdata/logic_test/select_index: 111 statements/queries
--- done: testdata/logic_test/select_index with config parallel-stmts: 133 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/timetz
--- progress: testdata/logic_test/sequences: 152 statements/queries
--- done: testdata/logic_test/upsert with config parallel-stmts: 90 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/statement_source
--- done: testdata/logic_test/timetz with config parallel-stmts: 58 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/table
--- FAIL: test/TestLogic/distsql-metadata/distinct_on (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/distinct_on
--- FAIL: test/TestLogic/distsql-metadata/sequences (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/sequences
--- FAIL: test/TestLogic/default/generators/expandArray (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/select (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/select
--- FAIL: test/TestLogic/default/family (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/family
--- FAIL: test/TestLogic/default/cascade/DeleteSetNull_ToUpdateCascade (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/array (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/array
--- FAIL: test/TestLogic/distsql-metadata/rename_database (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/rename_database
--- FAIL: test/TestLogic/distsql-metadata/snapshot_timestamp_drift (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/snapshot_timestamp_drift
--- FAIL: test/TestLogic/5node-distsql-metadata/distsql_stats (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql-metadata/distsql_stats
--- FAIL: test/TestLogic/default/serial (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/serial
--- FAIL: test/TestLogic/parallel-stmts/fk (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/fk
--- FAIL: test/TestLogic/distsql/collatedstring_index1 (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/collatedstring_index1
--- FAIL: test/TestLogic/parallel-stmts/rename_sequence (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/rename_sequence
--- FAIL: test/TestLogic/parallel-stmts/subquery (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/subquery
--- FAIL: test/TestLogic/parallel-stmts/tuple/tuple_placeholders (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/tuple with config parallel-stmts: 59 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/system
--- FAIL: test/TestLogic/distsql/suboperators (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/suboperators
--- FAIL: test/TestLogic/distsql-metadata/explain_types (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/explain_types
--- FAIL: test/TestLogic/default/inet (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/inet
--- FAIL: test/TestLogic/distsql/tuple/tuple_placeholders (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/tuple with config distsql: 59 tests, 0 failures
--- done: testdata/logic_test/timetz with config distsql: 58 tests, 0 failures
=== CONT  TestLogic/distsql/system
=== CONT  TestLogic/distsql/subquery
--- FAIL: test/TestLogic/distsql/cascade/DeleteSetNull_ToUpdateCascade (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/DeleteSetDefault_Basic1_WrongDefault (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/dangerous_statements (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/dangerous_statements
--- FAIL: test/TestLogic/distsql/cascade/DeleteCascade_Basic (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/backup with config distsql: 2 tests, 0 failures
=== CONT  TestLogic/distsql/postgresjoin
--- progress: testdata/logic_test/computed: 140 statements/queries
--- done: testdata/logic_test/computed with config distsql: 140 tests, 0 failures
--- done: testdata/logic_test/alias_types with config distsql: 10 tests, 0 failures
=== CONT  TestLogic/distsql/postgres_jsonb
=== CONT  TestLogic/distsql/poison_after_push
--- progress: testdata/logic_test/case_sensitive_names: 66 statements/queries
--- done: testdata/logic_test/case_sensitive_names with config distsql: 80 tests, 0 failures
=== CONT  TestLogic/distsql/planhook
--- FAIL: test/TestLogic/distsql/cascade/DeleteCascade_DoubleSelfReference (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/create_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/create_index
--- FAIL: test/TestLogic/parallel-stmts/create_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/create_index
--- FAIL: test/TestLogic/default/drop_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/drop_index
--- FAIL: test/TestLogic/distsql/deep_interleaving (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/deep_interleaving
--- FAIL: test/TestLogic/parallel-stmts/run_control (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/run_control
--- FAIL: test/TestLogic/parallel-stmts/scrub (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/scrub
--- FAIL: test/TestLogic/5node-distsql/distsql_union (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql/distsql_union
--- FAIL: test/TestLogic/5node-distsql/distsql_union/Intersect (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/postgres_jsonb (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/postgres_jsonb
--- FAIL: test/TestLogic/distsql/family (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/family
--- FAIL: test/TestLogic/distsql-metadata/cascade/UpdateCascade_Basic (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/show_trace/autocommit (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/stars_in_subexpressions with config default: 12 tests, 0 failures
=== CONT  TestLogic/default/snapshot_unrelated_update
--- FAIL: test/TestLogic/distsql/alias_types (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/alias_types
--- FAIL: test/TestLogic/parallel-stmts/snapshot_issue2861 (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/snapshot_issue2861
--- FAIL: test/TestLogic/default/create_statements (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/create_statements
--- FAIL: test/TestLogic/parallel-stmts/pg_catalog (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/pg_catalog
--- FAIL: test/TestLogic/distsql-metadata/cascade/DeleteSetDefault_Basic2 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/computed (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/computed
--- FAIL: test/TestLogic/default/serializable_eager_restart (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/serializable_eager_restart
--- FAIL: test/TestLogic/distsql/cascade/UpdateSetNull_ToUpdateCascade (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/cascade/DeleteCascade_Basic (0.000s)
Test ended in panic.

------- Stdout: -------
E180514 21:15:46.516284 512776 sql/distsqlrun/server.go:391  [client=127.0.0.1:36758,user=root,n1] error setting up flow: error type checking constant value: could not parse "a" as type int: strconv.ParseInt: parsing "a": invalid syntax
--- FAIL: test/TestLogic/parallel-stmts/drop_sequence (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/drop_sequence
--- FAIL: test/TestLogic/default/distsql_expr (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/distsql_expr
--- FAIL: test/TestLogic/distsql/generators/nested_SRF (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/DeleteSetDefault_Basic1 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/cascade/UpdateCascade_InterleavedRestrict (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/5node-distsql-metadata/interleaved_join (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql-metadata/interleaved_join
--- FAIL: test/TestLogic/default/select_search_path (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/select_search_path
--- FAIL: test/TestLogic/parallel-stmts/rename_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/rename_index
--- FAIL: test/TestLogic/parallel-stmts/cascade/UpdateSetDefault_ToUpdateCascadeNotNull (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/DeleteSetDefault_ToUpdateCascade#01 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/errors (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/errors
--- FAIL: test/TestLogic/distsql-metadata/cascade/UpdateSetDefault_Basic2 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/UpdateSetDefault_ToUpdateCascadeNotNull (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/optimizer (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/optimizer
--- FAIL: test/TestLogic/parallel-stmts/manual_retry/automatic_retry#01 (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/privileges_table with config parallel-stmts: 87 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/inverted_index
--- done: testdata/logic_test/no_primary_key with config parallel-stmts: 17 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/interleaved
--- done: testdata/logic_test/pg_catalog with config parallel-stmts: 118 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/insert
--- done: testdata/logic_test/json_builtins with config parallel-stmts: 175 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/inet
--- done: testdata/logic_test/json with config parallel-stmts: 103 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/generators
--- done: testdata/logic_test/inet with config parallel-stmts: 172 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/float
--- FAIL: test/TestLogic/distsql/cascade/UpdateSetNull_ToUpdateCascadeNotNull (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/cascade/UpdateCascade_CompositeFKs (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/5node-distsql-disk/distsql_numtables (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql-disk/distsql_numtables
--- FAIL: test/TestLogic/distsql/explain (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/explain
--- FAIL: test/TestLogic/distsql-metadata/name_escapes (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/name_escapes
--- FAIL: test/TestLogic/default/default (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/default
--- FAIL: test/TestLogic/distsql/snapshot_certain_read (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/snapshot_certain_read
--- FAIL: test/TestLogic/distsql-metadata/cascade/UpdateSetNull_Basic1 (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/alter_table with config distsql-metadata: 165 tests, 0 failures
--- total progress: 28977 statements/queries
--- FAIL: test/TestLogic/default/distinct_on (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/distinct_on
--- FAIL: test/TestLogic/default/function_lookup (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/function_lookup
--- FAIL: test/TestLogic/default/insert (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/insert
--- FAIL: test/TestLogic/default/builtin_function (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/builtin_function
--- FAIL: test/TestLogic/distsql-metadata/snapshot_issue2861 (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/snapshot_issue2861
--- FAIL: test/TestLogic/distsql-metadata/generators/nested_SRF (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/cascade/UpdateCascade_TwoUpdatesReverse (0.000s)
Test ended in panic.

------- Stdout: -------
--- progress: testdata/logic_test/alter_table: 145 statements/queries
--- FAIL: test/TestLogic/distsql/check_constraints (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/check_constraints
--- FAIL: test/TestLogic/distsql-metadata/float (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/float
--- FAIL: test/TestLogic/default/cascade/DeleteCascade_CompositeFKs (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/array with config default: 233 tests, 0 failures
--- FAIL: test/TestLogic/default/cascade/UpdateCascade_Interleaved (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/UpdateCascade_TwoUpdates (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/cascade/DeleteSetDefault_ToUpdateCascadeNotNull (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/rename_sequence (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/rename_sequence
--- FAIL: test/TestLogic/parallel-stmts/postgres_jsonb (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/postgres_jsonb
--- FAIL: test/TestLogic/distsql-metadata/poison_after_push (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/poison_after_push
--- FAIL: test/TestLogic/default/scatter (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/scatter
--- FAIL: test/TestLogic/parallel-stmts/statement_source (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/statement_source
--- FAIL: test/TestLogic/parallel-stmts/update (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/update
--- FAIL: test/TestLogic/parallel-stmts/cascade/DeleteSetDefault_Basic2 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/cascade (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/cascade
--- FAIL: test/TestLogic/distsql/inverted_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/inverted_index
--- FAIL: test/TestLogic/5node-distsql-disk/distsql_distinct_on (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql-disk/distsql_distinct_on
--- FAIL: test/TestLogic/default/alter_table (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/alter_table
--- FAIL: test/TestLogic/distsql/cascade/DeleteSetDefault_ToUpdateCascade (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/json (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/json
--- FAIL: test/TestLogic/distsql-metadata/txn (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/txn
--- FAIL: test/TestLogic/parallel-stmts/cascade/DeleteSetDefault_Basic1_WrongDefault (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/scrub (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/scrub
--- FAIL: test/TestLogic/distsql/generators/24866 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/parallel-stmts/with (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/with
=== CONT  TestLogic/parallel-stmts/serializable_eager_restart
=== CONT  TestLogic/parallel-stmts/returning
=== CONT  TestLogic/parallel-stmts/serial
=== CONT  TestLogic/parallel-stmts/sequences
=== CONT  TestLogic/parallel-stmts/select_non_covering_index_filtering
=== CONT  TestLogic/parallel-stmts/select_non_covering_index
=== CONT  TestLogic/parallel-stmts/select_index_hints
=== CONT  TestLogic/parallel-stmts/select_index
--- done: testdata/logic_test/select_non_covering_index with config parallel-stmts: 18 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/select
--- done: testdata/logic_test/serial with config parallel-stmts: 11 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/scrub
--- done: testdata/logic_test/returning with config parallel-stmts: 5 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/schema_change_retry
--- done: testdata/logic_test/select_non_covering_index_filtering with config parallel-stmts: 10 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/scale
--- done: testdata/logic_test/serializable_eager_restart with config parallel-stmts: 11 tests, 0 failures
--- done: testdata/logic_test/select_index_hints with config parallel-stmts: 28 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/run_control
=== CONT  TestLogic/parallel-stmts/time
--- done: testdata/logic_test/run_control with config parallel-stmts: 28 tests, 0 failures
--- total progress: 8717 statements/queries
=== CONT  TestLogic/parallel-stmts/with
--- done: testdata/logic_test/time with config parallel-stmts: 59 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/window
--- done: testdata/logic_test/scale with config parallel-stmts: 50 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/where
--- done: testdata/logic_test/schema_change_retry with config parallel-stmts: 12 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/views
--- done: testdata/logic_test/scrub with config parallel-stmts: 32 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/uuid
--- done: testdata/logic_test/select with config parallel-stmts: 107 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/upsert
--- done: testdata/logic_test/with with config parallel-stmts: 19 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/update
--- done: testdata/logic_test/where with config parallel-stmts: 19 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/txn
--- done: testdata/logic_test/window with config parallel-stmts: 167 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/tuple
--- done: testdata/logic_test/uuid with config parallel-stmts: 29 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/truncate
--- progress: testdata/logic_test/select_index: 111 statements/queries
--- done: testdata/logic_test/select_index with config parallel-stmts: 133 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/timetz
--- progress: testdata/logic_test/sequences: 152 statements/queries
--- done: testdata/logic_test/upsert with config parallel-stmts: 90 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/statement_source
--- done: testdata/logic_test/timetz with config parallel-stmts: 58 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/table
--- FAIL: test/TestLogic/distsql-metadata/distinct_on (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/distinct_on
--- FAIL: test/TestLogic/distsql-metadata/sequences (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/sequences
--- FAIL: test/TestLogic/default/generators/expandArray (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/select (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/select
--- FAIL: test/TestLogic/default/family (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/family
--- FAIL: test/TestLogic/default/cascade/DeleteSetNull_ToUpdateCascade (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/array (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/array
--- FAIL: test/TestLogic/distsql-metadata/rename_database (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/rename_database
--- FAIL: test/TestLogic/distsql-metadata/snapshot_timestamp_drift (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/snapshot_timestamp_drift
--- FAIL: test/TestLogic/5node-distsql-metadata/distsql_stats (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql-metadata/distsql_stats
--- FAIL: test/TestLogic/default/serial (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/serial
--- FAIL: test/TestLogic/parallel-stmts/fk (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/fk
--- FAIL: test/TestLogic/distsql/collatedstring_index1 (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/collatedstring_index1
--- FAIL: test/TestLogic/parallel-stmts/rename_sequence (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/rename_sequence
--- FAIL: test/TestLogic/parallel-stmts/subquery (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/subquery
--- FAIL: test/TestLogic/parallel-stmts/tuple/tuple_placeholders (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/tuple with config parallel-stmts: 59 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/system
--- FAIL: test/TestLogic/distsql/suboperators (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/suboperators
--- FAIL: test/TestLogic/distsql-metadata/explain_types (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql-metadata/explain_types
--- FAIL: test/TestLogic/default/inet (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/inet
--- FAIL: test/TestLogic/distsql/tuple/tuple_placeholders (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/tuple with config distsql: 59 tests, 0 failures
--- done: testdata/logic_test/timetz with config distsql: 58 tests, 0 failures
=== CONT  TestLogic/distsql/system
=== CONT  TestLogic/distsql/subquery
--- FAIL: test/TestLogic/distsql/cascade/DeleteSetNull_ToUpdateCascade (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/DeleteSetDefault_Basic1_WrongDefault (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/dangerous_statements (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/dangerous_statements
--- FAIL: test/TestLogic/distsql/cascade/DeleteCascade_Basic (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/backup with config distsql: 2 tests, 0 failures
=== CONT  TestLogic/distsql/postgresjoin
--- progress: testdata/logic_test/computed: 140 statements/queries
--- done: testdata/logic_test/computed with config distsql: 140 tests, 0 failures
--- done: testdata/logic_test/alias_types with config distsql: 10 tests, 0 failures
=== CONT  TestLogic/distsql/postgres_jsonb
=== CONT  TestLogic/distsql/poison_after_push
--- progress: testdata/logic_test/case_sensitive_names: 66 statements/queries
--- done: testdata/logic_test/case_sensitive_names with config distsql: 80 tests, 0 failures
=== CONT  TestLogic/distsql/planhook
--- FAIL: test/TestLogic/distsql/cascade/DeleteCascade_DoubleSelfReference (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/create_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/create_index
--- FAIL: test/TestLogic/parallel-stmts/create_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/create_index
--- FAIL: test/TestLogic/default/drop_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/drop_index
--- FAIL: test/TestLogic/distsql/deep_interleaving (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/deep_interleaving
--- FAIL: test/TestLogic/parallel-stmts/run_control (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/run_control
--- FAIL: test/TestLogic/parallel-stmts/scrub (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/scrub
--- FAIL: test/TestLogic/5node-distsql/distsql_union (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql/distsql_union
--- FAIL: test/TestLogic/5node-distsql/distsql_union/Intersect (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/postgres_jsonb (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/postgres_jsonb
--- FAIL: test/TestLogic/distsql/family (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/family
--- FAIL: test/TestLogic/distsql-metadata/cascade/UpdateCascade_Basic (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/show_trace/autocommit (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/stars_in_subexpressions with config default: 12 tests, 0 failures
=== CONT  TestLogic/default/snapshot_unrelated_update
--- FAIL: test/TestLogic/distsql/alias_types (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/alias_types
--- FAIL: test/TestLogic/parallel-stmts/snapshot_issue2861 (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/snapshot_issue2861
--- FAIL: test/TestLogic/default/create_statements (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/create_statements
--- FAIL: test/TestLogic/parallel-stmts/pg_catalog (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/pg_catalog
--- FAIL: test/TestLogic/distsql-metadata/cascade/DeleteSetDefault_Basic2 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/computed (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/computed
--- FAIL: test/TestLogic/default/serializable_eager_restart (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/serializable_eager_restart
--- FAIL: test/TestLogic/distsql/cascade/UpdateSetNull_ToUpdateCascade (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql-metadata/cascade/DeleteCascade_Basic (0.000s)
Test ended in panic.

------- Stdout: -------
E180514 21:15:46.516284 512776 sql/distsqlrun/server.go:391  [client=127.0.0.1:36758,user=root,n1] error setting up flow: error type checking constant value: could not parse "a" as type int: strconv.ParseInt: parsing "a": invalid syntax
--- FAIL: test/TestLogic/parallel-stmts/drop_sequence (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/drop_sequence
--- FAIL: test/TestLogic/default/distsql_expr (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/distsql_expr
--- FAIL: test/TestLogic/distsql/generators/nested_SRF (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/DeleteSetDefault_Basic1 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/cascade/UpdateCascade_InterleavedRestrict (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/5node-distsql-metadata/interleaved_join (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/5node-distsql-metadata/interleaved_join
--- FAIL: test/TestLogic/default/select_search_path (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/select_search_path
--- FAIL: test/TestLogic/parallel-stmts/rename_index (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/parallel-stmts/rename_index
--- FAIL: test/TestLogic/parallel-stmts/cascade/UpdateSetDefault_ToUpdateCascadeNotNull (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/DeleteSetDefault_ToUpdateCascade#01 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/distsql/errors (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/distsql/errors
--- FAIL: test/TestLogic/distsql-metadata/cascade/UpdateSetDefault_Basic2 (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/cascade/UpdateSetDefault_ToUpdateCascadeNotNull (0.000s)
Test ended in panic.

--- FAIL: test/TestLogic/default/optimizer (0.000s)
Test ended in panic.

------- Stdout: -------
=== PAUSE TestLogic/default/optimizer
--- FAIL: test/TestLogic/parallel-stmts/manual_retry/automatic_retry#01 (0.000s)
Test ended in panic.

------- Stdout: -------
--- done: testdata/logic_test/privileges_table with config parallel-stmts: 87 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/inverted_index
--- done: testdata/logic_test/no_primary_key with config parallel-stmts: 17 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/interleaved
--- done: testdata/logic_test/pg_catalog with config parallel-stmts: 118 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/insert
--- done: testdata/logic_test/json_builtins with config parallel-stmts: 175 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/inet
--- done: testdata/logic_test/json with config parallel-stmts: 103 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/generators
--- done: testdata/logic_test/inet with config parallel-stmts: 172 tests, 0 failures
=== CONT  TestLogic/parallel-stmts/float
--- FAIL: test/TestLogic/distsql/cascade/UpdateSetNull_ToUpdateCascadeNotNull (0.000s)
Test ended in panic.


```

Please assign, take a look and update the issue accordingly.

--- a/Makefile
+++ b/Makefile
@@ -52,7 +52,7 @@ FILES :=
 SUBTESTS :=
 
 ## Test timeout to use for regular tests.
-TESTTIMEOUT := 4m
+TESTTIMEOUT := 8m
 
 ## Test timeout to use for race tests.
 RACETIMEOUT := 25m

